app/assets/javascripts/classes/GraphVM.ts

```diff
@@ -43,9 +43,11 @@ class GraphVM implements StatisticView {
     started: boolean = false;
     container: HTMLElement = null;
     data: Series[] = [];
+    buffer: number[][][] = [];
```

Comment:
Series may not be the right object, in fact there may not be one right now at all, but using raw number here may create problems in the future. Another option why not put the buffer into data and and make methods to modify the pause/unpaused state of the Series.